CodeStarterCamp_Week2/STEP2.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  STEP2.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 한겨레 on 2022/06/29.
+//
+
+import Foundation
+
+//로또 번호 추첨
+func makeLottoNumbers() -> Set<Int> {
+    
+    var lottoNumbers: Set<Int> = Set<Int>()
+
+    while lottoNumbers.count<6 {
+        lottoNumbers.insert(Int.random(in: 1...45))
+    }
+    return lottoNumbers
+}
```

Comment:
또,

```swift
print(makeLottoNumbers())
```
를 실행하면, 랜덤의 숫자 6개가 나오는 걸 확인할 수 있어요!
랜덤의 6개의 숫자를 내림차순으로 보고싶다면 어떤 작업을 진행하면 될까요?

(swift 배열 정렬 검색하기)

Reply:
`print(makeLottoNumbers()).sorted(by: ) `를 사용하여 내림차순으로 볼 거 같습니다!!

Reply:
검색하고 응용하는 모습 최고네요 ㅎㅎ

```suggestion
func makeLottoNumbers() -> Set<Int> {
    
    var lottoNumbers: Set<Int> = Set<Int>()

    while lottoNumbers.count<6 {
        lottoNumbers.insert(Int.random(in: 1...45))
    }
    return lottoNumbers.sort()
}
```

이렇게 함수 내부에서 적용한다면

`print(makeLottoNumbers())` 로도 내림차순을 받을 수 있겠죠?